urbs/input.py

```diff
@@ -134,6 +134,10 @@ def pyomo_model_prep(data, timesteps):
     m.r_out_min_fraction = m.r_out_min_fraction['ratio-min']
     m.r_out_min_fraction = m.r_out_min_fraction[m.r_out_min_fraction > 0]
 
+    # storages with initial state as variable
+    m.stor_init_bound = m.storage['init']
+    m.stor_init_bound = m.stor_init_bound[m.stor_init_bound >= 0]
```

Comment:
Small suggestion: pop up error in validation file when "init" >1 (just like supim).  Apart from that, I think the change can be merged.